src/k-provider/ovp/loaders/custom-data-loader.js

```diff
@@ -0,0 +1,56 @@
+//@flow
+import RequestBuilder from '../../../util/request-builder';
+import OVPConfiguration from '../config';
+
+export default class OVPCustomDataLoader implements ILoader {
+  _requests: Array<RequestBuilder>;
+  _response: any = {};
+  _pluginName: string;
+
+  static get id(): string {
+    return 'custom';
+  }
+
+  /**
+   * @constructor
+   * @param {Object} params loader params
+   */
+  constructor(params: Object) {
+    const config = OVPConfiguration.get();
+    this.requests = params.requests.map(request => ({
+      ...request,
+      url: request.getUrl(config.serviceUrl),
+      params: {...request.params, ks: params.ks}
+    }));
```

Comment:
seems like we have a convention for this to be set inside a method called `buildRequests` so lets keep it

Reply:
Not relevant anymore as the entire Loader was taken out the providers and moved to dual screen